warp/src/main/java/com/schibsted/nmp/warp/components/legacy/WarpTextFieldView.kt

```diff
@@ -0,0 +1,126 @@
+package com.schibsted.nmp.warp.components.legacy
+
+import android.content.Context
+import android.util.AttributeSet
+import androidx.compose.material3.Icon
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.saveable.rememberSaveable
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.platform.AbstractComposeView
+import androidx.compose.ui.res.painterResource
+import androidx.core.content.withStyledAttributes
+import com.schibsted.nmp.warp.R
+import com.schibsted.nmp.warp.components.WarpTextField
+import com.schibsted.nmp.warp.components.ext.getTextFromIdOrString
+
+
+class WarpTextFieldView @JvmOverloads constructor(
+    context: Context,
+    attrs: AttributeSet? = null,
+    defStyle: Int = 0
+) : AbstractComposeView(context, attrs, defStyle) {
+
+    private var textFieldEnabled = true
```

Comment:
These should all be public right? What is the point of disposeComposition otherwise?

Reply:
Yep, my bad attention mistake